conda/plugins/subcommands/doctor/__init__.py

```diff
@@ -39,6 +39,8 @@ def execute(args: argparse.Namespace) -> None:
     from .health_checks import display_health_checks
 
     display_health_checks(context.target_prefix, verbose=context.verbose)
+    for health_check in context.plugin_manager.get_health_checks():
+        health_check.action
```

Comment:
```suggestion
        health_check.action()
```

If this is callable, then it needs to be invoked 😉.

Reply:
I was getting this error:

```      
File "/Users/maheiramkhan/documents/github/conda/conda/plugins/subcommands/doctor/__init__.py", line 43, in execute
        health_check.action()
    TypeError: 'NoneType' object is not callable
```